app/controllers/light/users_controller.rb

```diff
@@ -55,47 +49,47 @@ def unsubscribe
 
     def subscribe
       if @user.present? && @user.sidekiq_status == 'Unsubscribed'
-        @user.update(is_subscribed: 'true',
-                    sidekiq_status: 'Subscribed',
-                    subscribed_at: DateTime.now,
-                    remote_ip: request.remote_ip,
-                    user_agent: request.env['HTTP_USER_AGENT'])
+        @user.update(
+          is_subscribed: 'true',
```

Comment:
use boolean value instead of string 'true'

Reply:
Okay